virt-v2v/cold/scripts/rhel/run/network_config_util.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
```

Comment:
There are some comments in line below, but after some thingking I wonder if we can break this down into two functions:

One for sysV the next for NM, in future maybe one for systemd.

Thus for example:

```
udev_gen_ifcfg() {
  sed -e "s/^/S_HW=/" -e "s/\(:ip:\)\([^,]\+\).*/ S_IP=\2/" $V2V_MAP_FILE | while read HW_IP;
  do
    export $HW_IP ;
    IFCFG=$(grep $S_HW /etc/sysconfig/network-scripts/*)
    [[ -z "$IFCFG" ]] && return 1
    source $IFCFG
    echo "SUBSYSTEM==\"net\",ACTION==\"add\",ATTR{address}==\"$S_HW\",NAME=\"$DEVICE\""
  done
}

udev_gen_nm() { ... }

check_dupe_hws() {
  grep -io -E "[0-9A-F:]{17}" | sort | uniq -d && { echo "ERR: Duplicate hw" ; exit 2 ; }
}

main() {
  {
    udev_gen_ifcfg
    udev_gen_nm
  } | check_dupe_hws | write_udev
}
```

thus generally speaking using the map for each static ip configuration way separately, and in the end check if any macaddr was mentioned twice, and abort.